docs-2.0/nebula-explorer/operation-guide/ex-ug-canvas.md

```diff
@@ -5,26 +5,41 @@ This topic describes operation in canvas.
 ## Display vertexes and edges
 
 Move the mouse to the vertex or edge to view in detail. The following shows the detailed information of the vertex with `VID 107`:
+
 ![show](../figs/ex-ug-024-1.png)
 
 ## Batch selection
 
 Explorer supports batch selection and views the data of multiple vertexes and edges. The detailed data can be opened and viewed in the vertexes and edges overview at the lower-left corner of the canvas. It also supports exporting CSV files of selected vertexes or edges.
+
 ![review](../figs/ex-ug-027.png)
 
 ### Frame selection
 
 After clicking the ![frameselect](../figs/nav-frameSelect.png) icon, hold down the left button to drag and select multiple vertexes and edges. Examples are as follows:
-![slect](../figs/ex-ug-023-1.png)
+
+![select](../figs/ex-ug-023-1.png)
 
 ### Click to select multiple vertexes and edges
 
 Click the ![singleselect](../figs/nav-singleSelect.png) icon or hold down Shift, click and select multiple vertexes and edges with the mouse, and click the blank space to cancel the selection. Examples are as follows:
+
 ![select](../figs/ex-ug-025.png)
 
 ## Quick operation
 
 You can select one or more vertexes and edges, and right-click in the blank area to expand the vertexes, search the path between two vertexes, and show or hide their property on the canvas. The number of vertexes and edges you choose will affect the operations that can be performed. For more information, see [Graph Exploration Expansion](../operation-guide/ex-ug-graph-exploration.md).
+
 ![quick](../figs/ex-ug-026.png)
 
-Click **Fit Selection** to move the selected data to the center of the canvas for users to view.
\ No newline at end of file
+Click **Fit Selection** to move the selected data to the center of the canvas for users to view.
+
+## Vertex Filter
+
+You can filter the vertexes displayed on the canvas. In this example, select the vertex whose Tag is `player`,`age>=35` and click the `Apply Filter` button.
```

Comment:
[vertex whose Tag is `player`,`age>=35` and click the `Apply Filter` button.]

-->whose Tags are xxx and xxx, and then click xx.

Reply:
[filter the vertexes] -->filter the vertices